src/test/scala/org/broadinstitute/dsde/firecloud/mock/MockGoogleServicesDAO.scala

```diff
@@ -14,7 +14,13 @@ import scala.concurrent.{ExecutionContext, Future}
 
 class MockGoogleServicesDAO extends GoogleServicesDAO {
   override def getAdminUserAccessToken: String = ""
-  override def getBucketObjectAsInputStream(bucketName: String, objectKey: String): InputStream = new ByteArrayInputStream("firecloud-dev".getBytes("UTF-8"))
+  override def getBucketObjectAsInputStream(bucketName: String, objectKey: String): InputStream = {
```

Comment:
This can throw an `IOException` so I would like a key that will cause one, and tests for how we handle it